### PR TITLE
Webpack compiler

### PR DESCRIPTION
**Pull request by nervewax**
Not 100% finished, but ready to test. It might be worth keeping a seperate branch with grunt on it for backup and legacy purposes?

- Would be nice to have webpack look at an index.js inside theme, core-plugin and any other plugin, then take in all assets and output them.
- More testing with actual development to see if this is fast enough and if it works correctly with edge cases

### CODE REVIEW

File: kapow-boilerplate/build/wp-content/themes/my-project/inc/enqueue-assets.php [hunk 1]
```diff
@@ -27,58 +27,62 @@ function kapow_enqueue_assets() {
 
 	global $asset_suffix;
 
-	// Header JS.
-	$header_js_url  = get_stylesheet_directory_uri() . '/assets/js/header' . $asset_suffix . '.js';
-	$header_js_path = dirname( MY_PROJECT_THEME_ROOT ) . '/assets/js/header' . $asset_suffix . '.js';
+	wp_enqueue_script( 'header', get_stylesheet_directory_uri() . '/assets/js/header.js', array(), null, false );
```
Comment (davetgreen) [inline kapow-boilerplate/build/wp-content/themes/my-project/inc/enqueue-assets.php]: Cache busting needs re-introducing in this file. @davetgreen to sort.

**Comment by davetgreen** [comment]
🚀 🐶
